xdsl/ir/affine/affine_map.py

```diff
@@ -47,6 +47,30 @@ def compose(self, map: AffineMap) -> AffineMap:
             results=results,
         )
 
+    def inverse_permutation(self) -> AffineMap | None:
```

Comment:
Can you add a comment that this will not work when the AffineMap has symbols, and the cases for which this returns None?

Reply:
Sure.